src/arviz_plots/plots/lm_plot.py

```diff
@@ -0,0 +1,412 @@
+        aes_by_visuals = {}
+    else:
+        aes_by_visuals = aes_by_visuals.copy()
+
+    if stats is None:
+        stats = {}
+    else:
+        stats = stats.copy()
+
+    if labeller is None:
+        labeller = BaseLabeller()
+    if line_kind is None:
+        line_kind = rcParams["stats.point_estimate"]
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    obs_data = get_group(dt, "observed_data")
+    if y is None:
+        y = list(obs_data.data_vars)[0]
+
+    if isinstance(y, xr.Dataset):
+        raise TypeError(
+            "y can't be a dataset because multiple target variables are not supported yet."
+        )
+
+    if not isinstance(y, xr.DataArray):
+        y = process_group_variables_coords(
+            dt, group="observed_data", var_names=y, filter_vars=filter_vars, coords=coords
+        )
+
+    const_data = get_group(dt, "constant_data")
+    if x is None:
+        x = list(const_data.data_vars)
+
+    if not isinstance(x, xr.DataArray | xr.Dataset):
+        x = process_group_variables_coords(
+            dt, group="constant_data", var_names=x, filter_vars=filter_vars, coords=coords
+        )
+
+    (target_var,) = y.data_vars
+    independent_var = list(x.data_vars)
+
+    if y_pred is None:
+        y_pred = target_var
+
+    if isinstance(y_pred, xr.Dataset):
+        raise TypeError(
+            "y_pred can't be a dataset because multiple target variables are not supported yet."
+        )
+
+    if not isinstance(y_pred, xr.DataArray):
+        y_pred = process_group_variables_coords(
+            dt,
+            group=group,
+            var_names=y_pred,
+            filter_vars=filter_vars,
+            coords=coords,
+        )
+
+    if x_pred is None:
+        x_pred = independent_var
+
+    if not isinstance(x_pred, xr.DataArray | xr.Dataset):
+        if group == "predictions":
+            x_pred = process_group_variables_coords(
+                dt,
+                group="predictions_constant_data",
+                var_names=x_pred,
+                filter_vars=filter_vars,
+                coords=coords,
+            )
+        else:
+            x_pred = x
+
+    if isinstance(ci_prob, Sequence):
+        x_with_prob = x.expand_dims(dim={"prob": ci_prob})
+    else:
+        x_with_prob = x
+
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    bg_color = plot_bknd.get_background_color()
+    _, contrast_gray_color = get_contrast_colors(bg_color=bg_color, gray_flag=True)
+    if plot_collection is None:
+        pc_kwargs.setdefault("cols", "__variable__")
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        if isinstance(ci_prob, Sequence):
+            alpha_dims = pc_kwargs["aes"].get("alpha", None)
+            if alpha_dims is None:
```

Comment:
we don't use alpha dims so we can update the condition to

```suggestion
            if "alpha" not in pc_kwargs["aes"]:
```